lms/static/sass/base/_grid-settings.scss

```diff
@@ -21,4 +21,4 @@ $edx-bp-tiny: new-breakpoint(max-width 320px 4);
 $edx-bp-small: new-breakpoint(min-width 321px max-width 540px, 4);
 $edx-bp-medium: new-breakpoint(min-width 541px max-width 768px, 8);
 $edx-bp-large: new-breakpoint(min-width $large-min-width 12);
-$edx-bp-huge:  new-breakpoint(min-width 980px 12);
+$edx-bp-huge:  new-breakpoint(min-width 980px 12);
```

Comment:
Nit: missing new line

Reply:
Be sure to ran stylelint against your files so that you are obeying our Sass guidelines:

https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/sass-guidelines.html